test/tests/redirects.test.js

```diff
@@ -0,0 +1,38 @@
+import { expect } from 'chai'
+import request from 'request'
+import { bootServer, mockProxy, mockApi } from '../utils'
+
+describe('Handling redirects', () => {
+
+  let tapestry = null
+  let config = {
+    redirectPaths: {
+      '/redirect/from/this-path': '/about/home'
```

Comment:
I did think that, but then I also thought.... what if you want 25000 redirects in a file, that'd  add a lot of extra file size.

Using `from` and `to` we could probably validate the config with joi too.

If you have multiple:

```
redirectPaths: [{
  from: '/',
  to: '/'
}, {
 from: '/',
  to: '/'
}, {
 from: '/',
  to: '/'
}, {
 from: '/',
  to: '/'
}, {
 from: '/',
  to: '/'
}, {
 from: '/',
  to: '/'
}]

I think 

{
  '/redirect/from/this-path': '/about/home',
  '/redirect/from/this-path': '/about/home',
  '/redirect/from/this-path': '/about/home',
  '/redirect/from/this-path': '/about/home',
  '/redirect/from/this-path': '/about/home',
  '/redirect/from/this-path': '/about/home'
}
```

Looks a bit neater? 

IDK, something we could bikeshed for a while... for an advanced config feature. I'm easy either way.